common/build.gradle

```diff
@@ -21,9 +32,9 @@ repositories {
 group 'com.trikot.sample'
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 30
     defaultConfig {
-        minSdkVersion 21
+        minSdkVersion 24
```

Comment:
Minimum for trikot is now Android 7?

Reply:
Well, the android one was at 24 already, I just unified the expectation? Or I might have misunderstood the impact of this value.

Reply:
I guess it should be consistent, but I'd keep the lowest value that works. Projects can still have an higher floor.
Most trikot libs seems to have 14 as minimum, so most likely patron should be consistent with that. Or at least not higher than 21 if possible.